### PR TITLE
Write documentation

### PR DESCRIPTION
**Pull request by daiyi**
- make it friendly for contributors! Wrote instructions on how to set up the project for development
- clarified steps for trace tool installation
- moved the installation instructions to the top because that's probably the most relevant to most people

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,3 +41,20 @@ By default, re-frame tracing is compiled out, so it won't impose a performance c
...
+### setting up re-frame-trace for development
+
+- Clone `re-frame-trace` to your machine.
+- Go into the root directory of a project you'd like to use as a host to test re-frame-trace with. For example, you can use the [todo-mvc](https://github.com/Day8/re-frame/tree/master/examples/todomvc) project.
+- Add re-frame-trace into this project using the [instructions](#getting-started) above.
+- Create a folder called `checkouts:`
+  ```
+  mkdir checkouts
+  ```
+- Create a symlink from your local re-frame-trace project in the checkouts folder:
```
Comment (danielcompton) [inline README.md]: I'd suggest cd'ing into the checkouts directory here, as it's easy to mess up the `ln` if you're not in the directory you're linking to. If you use a relative path in the link, it is relative to the link that you're creating, not the location you are at, but your tab completion will be relative to your current working directory.

File: README.md [hunk 2]
```diff
@@ -41,3 +41,20 @@ By default, re-frame tracing is compiled out, so it won't impose a performance c
...
+  ```
+  ln -s path/to/your/local/re-frame-trace checkouts/re-frame-trace
+  ```
```
Comment (danielcompton) [inline README.md]: We also need to mention that if you're using figwheel, you need to add the checkouts project to your :cljsbuild :source-paths.

File: README.md [hunk 3]
```diff
@@ -41,3 +41,20 @@ By default, re-frame tracing is compiled out, so it won't impose a performance c
 The [preloads](https://github.com/clojure/clojurescript/wiki/Compiler-Options#preloads) option (`:preloads [day8.re-frame.trace.preload]`) has to be set in order to automatically monkeypatch Reagent to add appropriate lifecycle hooks. Yes this is gross, and yes we will try and make a PR to reagent to add proper hooks, once we know exactly what we need. The preload namespace also injects a div containing the devtools panel into the DOM.
 
 Now you can start up your application. Once it is loaded, press Ctrl+H to slide open the trace panel and enable tracing. When the panel is closed, tracing is disabled.
+
+
+## development
```
Comment (danielcompton) [inline README.md]: Development

File: README.md [hunk 4]
```diff
@@ -41,3 +41,20 @@ By default, re-frame tracing is compiled out, so it won't impose a performance c
...
+
+### setting up re-frame-trace for development
```
Comment (danielcompton) [inline README.md]: Setting up

**Comment by daiyi** [comment]
I wrote instructions for adding the checkouts to :source-paths, and also edited some of the other sections to make it more clear, after reading through it a week later and finding past-me less than eloquent. (:

**Pull request merged**